test/environments/main/Manifest.toml

```diff
@@ -26,12 +26,6 @@ git-tree-sha1 = "a2a6a5fea4d6f730ec4c18a76d27ec10e8ec1c50"
 uuid = "187b0558-2788-49d3-abe0-74a17ed4e7c9"
 version = "1.0.0"
 
-[[DataStructures]]
-deps = ["InteractiveUtils", "OrderedCollections"]
-git-tree-sha1 = "b7720de347734f4716d1815b00ce5664ed6bbfd4"
-uuid = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
-version = "0.17.9"
```

Comment:
Removing DataStructures.jl gets rid of the method ambiguities and "hides" a potential bug in Aqua.jl https://github.com/JuliaTesting/Aqua.jl/issues/39